src/Files.App/Views/Properties/CompatibilityPage.xaml

```diff
@@ -24,9 +24,10 @@
 		</ResourceDictionary>
 	</vm:BasePropertiesPage.Resources>
 
-	<ScrollViewer>
-		<StackPanel Padding="12" Spacing="4">
+	<ScrollViewer x:Name="RootScrollViewer">
+		<StackPanel Padding="12" Spacing="8">
```

Comment:
Can you share a screenshot of the before and after?

Reply:
![image](https://user-images.githubusercontent.com/62196528/232265678-1528f23d-74d5-4e7e-b565-d993b9edf2e7.png)

Reply:
Can you revert this change (for now), it's out of scope and doesn't align with the spacing in the setting dialog.

Reply:
Reverted